src/python/visclaw/colormaps.py

```diff
@@ -149,8 +149,12 @@ def add_colormaps(colormaps, data_limits=[0.0,1.0], data_break=0.5,
                                                              int(N / 2) + N % 2)
     #norm = colors.BoundaryNorm(boundaries=bounds, ncolors=N)
 
-    # Use this norm for proportional pw linear with better tick mark locations:
-    norm = colors.TwoSlopeNorm(data_break, data_limits[0], data_limits[1])
+    try:
+        # Use this norm for proportional pw linear with better tick mark locations:
+        norm = colors.TwoSlopeNorm(data_break, data_limits[0], data_limits[1])
+    except:
+        # older matplotlib.colors did not have TwoSlopeNorm, revert to old:
+        norm = colors.BoundaryNorm(boundaries=bounds, ncolors=N)
```

Comment:
Might be good to catch only the specific error, whatever that was.

Reply:
Fixed.